versiondb/client/restore.go

```diff
@@ -0,0 +1,121 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"path/filepath"
+	"strconv"
+
+	"cosmossdk.io/errors"
+	protoio "github.com/gogo/protobuf/io"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/snapshots"
+	snapshottypes "github.com/cosmos/cosmos-sdk/snapshots/types"
+
+	"github.com/crypto-org-chain/cronos/versiondb"
+	"github.com/crypto-org-chain/cronos/versiondb/tsrocksdb"
+)
+
+// RestoreVersionDBCmd returns a command to restore a versiondb from local snapshot
+func RestoreVersionDBCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "restore-versiondb <height> <format>",
+		Short: "Restore initial versiondb from local snapshot",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := server.GetServerContextFromCmd(cmd)
+
+			height, err := strconv.ParseUint(args[0], 10, 63)
```

Comment:
```suggestion
			height, err := strconv.ParseUint(args[0], 10, 64)
```

Reply:
Because it's to be converted to signed int64, so I choose 63bits here